apps/dashboard/package.json

```diff
@@ -4,7 +4,7 @@
     "@popperjs/core": "^2.6.0",
     "@rails/ujs": "^6.1.0",
     "@rails/webpacker": "5.2.1",
-    "bootstrap": "^3.4.1",
+    "bootstrap": "5.0.0-beta1",
```

Comment:
Is this intentional? seems like this is what we would save for https://github.com/OSC/ondemand/pull/771

Reply:
@ericfranz Yeah, that'll be in a new PR. #845 is the source of truth for Webpacker. #780 and #771 should be closed and cherry picked into #845.